docs/vNext/tutorials_fabric_existing.md

```diff
@@ -0,0 +1,724 @@
+
+The benchmark configuration file may be provided in a yaml or json format: here we will use a yaml format. Please note that yaml files are case sensitive and all labels are in lowercase. 
+
+The benchmark configuration file has a single required stanza:
+
+``` bash
+test:
+```
+### Create File
+
+Under the **benchmarks** folder create a file called **myAssetBenchmark.yaml**. That is the file we will be populating.
+
+### Test Block
+
+First add a root level block named `test`. In this block add:
+
+- a `name` key with the value `my-asset-benchmark`
+- a `description` key with a short description as the value, in this example that is `test benchmark`
+- a literal block named `workers` that has the following keys and values:
+    - `type: local`
+    - `number: 2`
+
+It should look like this so far: 
+
+Also add a literal block named `rounds`. This block contains each benchmark test round that is to be run headed by a unique round label. Rounds may be used to benchmark different smart contract methods, or the same method in a different manner. 
+
+The `round` block contains the following: 
+
+- `label` - the unique header label to use for the round.
+- `description` - a description of the round being run.
+- `txDuration` - the specification of the test duration, in seconds
+- `rateControl` - a rate control type, with options. 
+- `workload` - the workload module to use, with arguments to pass to the module. All arguments passed are available as `roundArguments` within the workload module.
+
+This is how it should be populated: 
+
+``` bash
+rounds:
+    - label: queryAsset
+      description: Query asset benchmark
+      txDuration: 30
+      rateControl: 
+        type: fixed-load
+        opts:
+          transactionLoad: 2
+      workload:
+        module: benchmarks/workload/queryAssetBenchmarks.js
+        arguments:
+          assets: 10
+          contractID: fabcar
+```
+
+### Resulting File
+
+The benchmark configuration file should now be fully populated. 
+
+It should look like this:
+
+```bash
+test:
+    name: my-asset-benchmark
+    description: test benchmark
+    workers:
+      type: local
+      number: 2
+    rounds:
+      - label: queryAsset
+        description: Query asset benchmark
+        txDuration: 30
+        rateControl: 
+          type: fixed-load
+          opts:
+            transactionLoad: 2
+        workload:
+          module: benchmarks/workload/queryAssetBenchmarks.js
+          arguments:
+            assets: 10
+            contractID: fabcar
```

Comment:
contractId maybe to match the parameter expected by the connector ? Also no fabcar anymore